src/main/kotlin/com/github/rushyverse/rtf/commands/RTFCommand.kt

```diff
@@ -30,25 +30,54 @@ class RTFCommand(
     suspend fun register() {
         commandAPICommand("rtf") {
 
-            subcommand("spectate") {
-
-                withArguments(IntegerArgument("game"))
-
+            subcommand("create") {
+                withArguments(IntegerArgument("gameID"))
                 playerExecutor { player, args ->
                     val gameIndex = args[0] as Int
                     var game = games.getGame(gameIndex)
 
                     plugin.launch {
 
-                        if (game == null && gameIndex == 1) {
+                        if (game == null) {
                             game = games.createAndSave(gameIndex)
+                        } else {
+                            player.sendMessage("game.already.exists")
                         }
 
                         game?.clientSpectate(clients.getClient(player) as ClientRTF)
                     }
                 }
             }
 
+            subcommand("list") {
+                playerExecutor { player, _ ->
+                    val length = games.games.size
+                    player.sendMessage("List of games ($length):")
+                    for (game in games.games) {
+                        player.sendMessage("#${game.id} - ${game.players.size}/${game.config.game.maxGames} - ${game.state()}")
+                    }
+                }
+            }
+
+            subcommand("spectate") {
+                withArguments(IntegerArgument("gameID"))
+                playerExecutor { player, args ->
+                    val gameIndex = args[0] as Int
+                    var game = games.getGame(gameIndex)
+
+                    if (game == null) {
+                        player.sendMessage("game.not.exists")
+                    } else {
+                        plugin.launch {
+
+                            game = games.createAndSave(gameIndex)
```

Comment:
The 'spectate' subcommand is not correct because it re-creates an already existing game. You can delete this line.
This means that it will be necessary to review the game creation logic from the main Hub menu.